src/assets/SearchIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+// tslint:disable:max-line-length
+// tslint:disable-next-line:variable-name
+const SvgComponent = () => (
```

Comment:
i see this one is an actual react component that renders and svg, the others are just svg files. i'm not sure why they aren't all done the same way